src/nupic/algorithms/Cells4.cpp

```diff
@@ -3085,63 +3095,16 @@ void Cells4::computeForwardPropagation(CState &state) {
   // Compute cell and segment activity by following forward propagation
   // links from each source cell.  _cellActivity will be set to the total
   // activity coming into a cell.
-#ifdef NTA_ARCH_64
-  const UInt multipleOf8 = 8 * (_nCells / 8);
```

Comment:
removed. Example of premature optimization! The generic solution is same/better and much easier to understand.